resources/css/app.css

```diff
@@ -0,0 +1 @@
+ @import "../sass/app.scss"
```

Comment:
if all this is going to do, better to remove this entry and only use the sass folder?